pkg/alicloud/apis/alicloud_provider_spec.go

```diff
@@ -29,13 +31,13 @@ type ProviderSpec struct {
 	DataDisks               []AlicloudDataDisk      `json:"dataDisks,omitempty"`
 	InstanceChargeType      string                  `json:"instanceChargeType,omitempty"`
 	InternetChargeType      string                  `json:"internetChargeType,omitempty"`
-	InternetMaxBandwidthIn  *int32                  `json:"internetMaxBandwidthIn,omitempty"`
-	InternetMaxBandwidthOut *int32                  `json:"internetMaxBandwidthOut,omitempty"`
+	InternetMaxBandwidthIn  *int                    `json:"internetMaxBandwidthIn,omitempty"`
+	InternetMaxBandwidthOut *int                    `json:"internetMaxBandwidthOut,omitempty"`
 	SpotStrategy            string                  `json:"spotStrategy,omitempty"`
 	IoOptimized             string                  `json:"IoOptimized,omitempty"`
 	Tags                    map[string]string       `json:"tags,omitempty"`
 	KeyPairName             string                  `json:"keyPairName"`
-	SecretRef               *corev1.SecretReference `json:"secretRef,omitempty"`
+	//SecretRef               *corev1.SecretReference `json:"secretRef,omitempty"`
```

Comment:
```suggestion
```

Reply:
I see machineClass already contains the `SecretRef` and `CredentialsSecretRef`, so the SecretRef in provider spec is not needed any more?

Reply:
Yes that is right. This field is not needed anymore as all the secret data is transported via the createRequest. Refer - https://github.com/gardener/machine-controller-manager-provider-alicloud/blob/master/vendor/github.com/gardener/machine-controller-manager/pkg/util/provider/driver/driver.go#L52-L53. And this secret as both the fields from the two secrets merged.